src/compiler/commandLineParser.ts

```diff
@@ -813,6 +813,9 @@ const commandOptionsWithoutBuild: CommandLineOption[] = [
     {
         name: "verbatimModuleSyntax",
         type: "boolean",
+        affectsEmit: true,

```

Comment:
Does this belong in its own PR? Looks unrelated

Reply:
This was related to baseline i fixed so added it here.

Reply:
As part of 1012104 and c871a27